src/components/qa/exercise-data-filter.cjsx

```diff
@@ -0,0 +1,29 @@
+_ = require 'underscore'
+
+{EcosystemsStore} = require '../../flux/ecosystems'
+
+BOOK_MAPPINGS =
+  '02040312-72c8-441e-a685-20e9333f3e1d': 'stax-soc'
```

Comment:
This part seems duplicated; could it be in 1 place and just pulled in from the flux store?